tests/js-modules/modules.py

```diff
@@ -117,37 +120,38 @@ def generate_and_verify_jwk(client):
         assert body["pem"] == pub_pem
 
     # EdDSA
-    priv_pem, pub_pem = infra.crypto.generate_eddsa_keypair()
+    # Note: x25519 is not supported by jwcrypto just yet
```

Comment:
See https://github.com/latchset/jwcrypto/blob/41fb08a00ad2a36a1d85bf77ad973b31144ef9f2/jwcrypto/jwk.py#L965 and https://github.com/latchset/jwcrypto/blob/41fb08a00ad2a36a1d85bf77ad973b31144ef9f2/jwcrypto/jwk.py#L81